### ISSUE TITLE
include Searchable breaks db:setup in production

### ISSUE THREAD

**Issue by milovanderlinden** [consul/consul#3293]
Today, I spend 4 hours trying to figure out why my db:setup and migrations will not run in `RAILS_ENV=production` mode. I have narrowed it down:

If a model is set up like
```ruby
class SomeModel < ActiveRecord::Base
  ...
  include Searchable
  ...
```

Having the `include Searchable` generates a query to the database. This will fail if the database **has no tables** (which is the case during db:setup)

Affected models:
- debate.rb
- proposal.rb
- questionrb
- legislation/proposal.rb
- spending_proposal.rb

For now I have commented the lines, this results in db:setup running correctly again.
But I will also look at the Searchable to see **WHY** it generates a query.

**Comment by milovanderlinden** [comment]
If I replace 
```ruby
include Searchable
```

with

```ruby
include PgSearch
```

Everything keeps working correct. So the error is not in the underlying PgSearch module.

**Comment by milovanderlinden** [comment]
And if I replace

```ruby
include Searchable
```

With 
```ruby
  pg_search_scope(
    :search,
    against: %i(
      description
      manufacturer_name
      name
    ),
    using: {
      tsearch: {
        dictionary: "english",
      }
    }
  )
```

It also remains working.

**Comment by javierm** [comment]
Closed via #3414.
